hack/test-integration.sh

```diff
@@ -78,9 +78,15 @@ function exectest() {
 
 	if [[ ${result} -eq 0 ]]; then
 		os::text::print_green "ok      $1"
-		exit 0
+		# Remove the etcd directory to free up space when the test case finished
+		# succesfully.
+		rm -rf /tmp/etcd????????? && exit 0
```

Comment:
> @deads2k @smarterclayton will something ugly like this do the job?

` || true` then keep `exit 0` separate?

Otherwise I'm ok cleaning up after ourselves like this.